command.go

```diff
@@ -99,8 +101,22 @@ func (c *Command) ProcConstructor(processConstructor ProcessConstructor) *Comman
 // been created by the Command for execution.
 func (c *Command) GetProcIDs() []string { return c.procIDs }
 
-// ApplyFromOpts uses the CreateOptions to configure the Command. If Args is
-// set on the CreateOptions, it will be ignored; the Args can be added using Add.
+// ApplyFromOpts uses the CreateOptions to configure the Command. If Args is set
+// on the CreateOptions, it will be ignored; the Args can be added using Add,
+// Append, AppendArgs, or Extend.
+// This overwrites options that were previously set in the following functions:
+// * AddEnv
+// * Environment
+// * RedirectErrorToOutput
+// * RedirectOutputToError
+// * SetCombinedSender
+// * SetErrorSender
+// * SetErrorWriter
+// * SetOutputOptions
+// * SetOutputSender
+// * SetOutputWriter
+// * SuppressStandardError
+// * SuppressStandardOutput
```

Comment:
I would just comma seperate this list, godoc formating isn't very smart.

Reply:
Done.